packages/ffe-cards-react/src/NewCard.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import classNames from 'classnames';
+import { oneOfType, node, func, string, bool } from 'prop-types';
+
+const NewCard = props => {
+    const {
+        className,
+        details,
+        element: Element,
+        heading,
+        icon,
+        condensed,
+        greyCharcoal,
+        ...rest
+    } = props;
+
+    return (
+        <Element
+            className={ classNames(
+                'ffe-new-card',
+                { 'ffe-new-card--condensed': condensed },
+                { 'ffe-new-card--grey-charcoal': greyCharcoal },
+                className
+            ) }
+            { ...rest }
+        >
+            {icon &&
+            React.cloneElement(icon, {
+                className: classNames(
+                    'ffe-new-card__icon',
+                    icon.props.className
+                ),
+            })}
+            <div className="ffe-new-card__body">
+                {heading && <p className="ffe-new-card__heading">{heading}</p>}
```

Comment:
Should the card heading be some kind of <h*> semantically?
If so, this needs to be decided by the consumer to make sure it's correct in their hierarchy of headings.

Reply:
i don't think so - it's not a heading as much as it is the text content of most cards.

Reply:
You can always accept either a `string` or a `node`

Reply:
If it shouldn't be a heading tag anyways, I think accepting only string is fine here.